Makefile

```diff
@@ -87,20 +90,22 @@ endif
 		sudo --preserve-env $(CONTAINERLAB) deploy --topo $(LAB_TOPOLOGY) --reconfigure && \
 		./scripts/deactivate_offloading.sh; fi
 
+.PHONY: external_network
+external_network:
+	@if ! docker network ls | grep -q mini_lab_ext; then \
+  		docker network create mini_lab_ext \
+			--driver=bridge \
+			--gateway=203.0.113.1 \
```

Comment:
Maybe add a reference to where this address block is described as for documentation: https://en.wikipedia.org/wiki/Reserved_IP_addresses